### ISSUE TITLE
publishToConfluence does not respect preambleTitle

### ISSUE THREAD

**Issue by rdmueller** [docToolchain/docToolchain#398]
see https://github.com/docToolchain/docker-image/issues/9

**Comment by rdmueller** [comment]
So, in this line, the right `preambleTitle` should be selected and set:

github.com/docToolchain/docToolchain/blob/eae7869a104d125abb3bba32a3347a6cbfa793ce/scripts/asciidoc2confluence.groovy#L706

and in this line it is used and the default `arc42`is set:

github.com/docToolchain/docToolchain/blob/eae7869a104d125abb3bba32a3347a6cbfa793ce/scripts/asciidoc2confluence.groovy#L734

somewhere around this, the problem has to be....

let me debug this tomorrow...

**Comment by dArignac** [comment]
Experiencing this as well. As you mentioned to debug it tomorrow (which indeed is in the past), any news on that?

**Comment by rdmueller** [comment]
thanx for the reminder. I guess it got lost in my backlog...
let's see...

**Comment by hbrands** [comment]
What has worked for me in the past is specifying the preambleTitle in the input, something like:

input = [[ file: "build/html5/arc42-template.html", ancestorId: "46956627", preambleTitle: "Architektur" ]]

perhaps this is a workaround for now?

**Comment by dArignac** [comment]
I rolled back to a commit where it was working for now, as a workaround.

**Comment by rdmueller** [comment]
sorry that I am too slow to respknd in a timely manner.

@hbrands thanx for the idea with the workaround

@dArignac wich version did you rollback to?

**Comment by rdmueller** [comment]
ok. I guess I found the problem - I was looking in the wrong place.
You use the docker image v1.1.0. So this is the script used
https://github.com/docToolchain/docToolchain/blob/v1.1.0/scripts/asciidoc2confluence.groovy
and it does not support the global `preableTitle`.

Since the docker image is somehow experimental, there is currently no image with a newer version of the repository...

Update: there is now a version rc-1.2.0 - this should fix the problem because it is based on the current source. Please give it a try and re-open this ticket if you experience any problems.

**Comment by dArignac** [comment]
Well, I created my own docker image inspired by your image, as we need it within our CI/CD as a Jenkins slave image. I used the `master` branch, that was when it failed with the `preambleTitle`. I then changed it to commit 7e547f3bf9a21d9b0b28324797341feda1a8e111 (that was the last time I pulled the DTC locally where it worked with the title) and it works.
I will review your changes within your docker image, to see what's new.
